app/queries/payments_query.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+class PaymentsQuery < BaseQuery
```

Comment:
I miss the test for this query object to be used a thorough definition of the query object... I'm not sure it behaves as it should be expected....

for example, the spec should include the validation, the scenario for payment requests where the payable is an invoice and when the payable is a payment request.